src/App.tsx

```diff
@@ -1,24 +1,368 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  FormEvent,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import cn from 'classnames';
+import { getAllTodos } from './api/todos';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
 import { UserWarning } from './UserWarning';
+import { Filters, Errors } from './utils/enums';
 
-const USER_ID = 0;
+const USER_ID = 10306;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedFilter, setSelectedFilter] = useState('All');
+  const [hasError, setHasError] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(Errors.NoErrors);
+  const [todoTitle, setTodoTitle] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [deletingTodoId, setDeletingTodoId] = useState<number | null>(null);
+
+  const delayErrorDisappear = useCallback(() => {
+    setTimeout(() => {
+      setHasError(false);
+    }, 3000);
+  }, []);
+
+  useEffect(() => {
+    getAllTodos(USER_ID)
+      .then(setTodos);
+  }, []);
+
+  const handleFilterSelection = useCallback((value: string) => {
```

Comment:
Remove `[selectedFilter]` from the useCallback dependencies array since it is not used inside the callback.